pynear/tests/integration/test_cli.py

```diff
@@ -0,0 +1,197 @@
+import json
+import os
+import random
+
+import delegator
+import pytest
+from retrying import retry
+
+from near.pynear.lib import NearLib
+
+nearlib = NearLib('http://localhost:3030/')
+
+
+@retry(stop_max_attempt_number=5, wait_fixed=1000)
+def check_devnet_health(process):
+    if not process.is_alive:
+        return False
+
+    return nearlib.check_health()
+
+
+@pytest.fixture
+def make_devnet(request):
+    def _make_devnet(base_dir):
```

Comment:
Will you also make sure that `healthz` corresponds to the DevNet being fully operational and not simply having its HTTP server up?

Reply:
`healthz` calls into view_latest_beacon_block which is the current check for health in other integration tests